sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 3.1.0-beta.1 (Unreleased)
-
```

Comment:
It will be good to also add that this version of the library now targets v2.1 of the service